functions/lib/stripe/webhooks.py

```
@@ -19,25 +18,41 @@
         else:
             self.isPrivate = isPrivate
+        if isinstance(completeFulfilmentSession, str):
+            self.completeFulfilmentSession = completeFulfilmentSession.lower() == "true"
+        else: 
+            self.completeFulfilmentSession = bool(completeFulfilmentSession)  # Treat empty strings as absent 
+            self.fulfilmentSessionId = fulfilmentSessionId
+            self.endFulfilmentEntityId = endFulfilmentEntityId
 
     def __post_init__(self):
         if not isinstance(self.eventId, str):
             raise ValueError("Event Id must be provided as a string.")
```

Ruff (0.13.1)

Prefer `TypeError` exception for invalid type

(TRY004)